scripts/acbuild-rkt-builder.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+set -ex
+
+MODIFY=${MODIFY:-""}
+FLAGS=${FLAGS:-""}
+IMG=${IMG:-"debian"}
+IMGVERSION=${VERSION:-"sid"}
+DOCKERIMG="$IMG:$VERSION"
+ACIFILE=${ACIFILE:-"$IMG-$IMGVERSION.aci"}
+OUTACI="rkt-builder.aci"
+BUILDDIR=/opt/build-rkt
+SRC_DIR=/opt/rkt
+ACI_GOPATH=/go
+VERSION=${VERSION:-"v0.10.0"}
+# if [ `expr match $VERSION '.*git$'` -gt 0 ]; then
+#     VERSION="$VERSION$(git rev-parse --short HEAD 2>/dev/null)$(git diff-index --quiet HEAD 2>/dev/null || echo -dirty)"
+# fi
+echo "Version: $VERSION"
+
+echo "Building $ACIFILE"
+
+which docker2aci || go get github.com/appc/docker2aci
+if ! [ -e "$ACIFILE" ]; then
+    docker2aci "docker://$DOCKERIMG"
+    # These base images dont always come with valid values
+    actool patch-manifest -user 0 -group 999999 --name $IMG-$IMGVERSION --exec /bin/bash --replace $ACIFILE
+    # Work around to make the image readable by acbuild:
+    # Put it through rkt CAS and acbuild will be able to untar it
+    BUILDER_IMG_ID="$(rkt fetch --insecure-skip-verify ./$ACIFILE)"
+    rm ./$ACIFILE
+    rkt image export $BUILDER_IMG_ID $ACIFILE
+fi
+
+# If modify is specified, pass the modify flag to each command and don't use
+# acbuild begin, write or end otherwise build with a context, and setup a trap
+# to handle failures,
+if ! [ -z "$MODIFY" ]; then
+    FLAGS="--modify $MODIFY $FLAGS"
+    OUTACI=$MODIFY
+else
+    trap "{ export EXT=$?; acbuild end && exit $EXT; }" EXIT
+    acbuild $FLAGS begin "$ACIFILE"
+fi
+
+acbuild $FLAGS set-name coreos.com/rkt/builder
+acbuild $FLAGS label add version $VERSION
+acbuild $FLAGS set-user 0
+acbuild $FLAGS set-group 99999
```

Comment:
Whoops. I used this because it was the UID i was seeing in other containers as the default group. I actually have no idea what's reasonable, I just know that a lot of it requires root.